
From: Nick Piggin <nickpiggin@yahoo.com.au>

Use CPU_DOWN_FAILED notifier in the sched-domains hotplug code.  This goes
with 4/8 "integrate cpu hotplug and sched domains"

Signed-off-by: Nick Piggin <nickpiggin@yahoo.com.au>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/kernel/sched.c |    1 +
 1 files changed, 1 insertion(+)

diff -puN kernel/sched.c~sched-use-cpu_down_failed-notifier kernel/sched.c
--- 25/kernel/sched.c~sched-use-cpu_down_failed-notifier	2004-10-05 02:09:49.879247536 -0700
+++ 25-akpm/kernel/sched.c	2004-10-05 02:09:49.885246624 -0700
@@ -4446,6 +4446,7 @@ static int update_sched_domains(struct n
 		return NOTIFY_OK;
 
 	case CPU_UP_CANCELED:
+	case CPU_DOWN_FAILED:
 	case CPU_ONLINE:
 	case CPU_DEAD:
 		/*
_
